examples/seq2seq/create_student.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import warnings
+from typing import List
+
+from torch import nn
+
+
+try:
+    from .distillation import LAYERS_TO_COPY
+except ImportError:
+    from distillation import LAYERS_TO_COPY
+
+from transformers import AutoModelForSeq2SeqLM, AutoTokenizer
+
+
+def get_layers_to_copy(n_student, n_teacher):
```

Comment:
Why can't we import all these helper functions?

Reply:
This assert fails for when you try, say 12-9 and `LAYERS_TO_SUPERVISE` is not needed here. Also the rest of the helper are methods so can't import them.
https://github.com/huggingface/transformers/blob/03fb8e79c68bc981bdcc2240257253c8f8596204/examples/seq2seq/distillation.py#L479

Reply:
Should we rather move them outside of the class ?